scripts/docker/tasking-manager/Dockerfile

```diff
@@ -35,7 +34,7 @@ RUN pip install --no-cache-dir pdm \
     && pdm install --prod --no-lock --no-editable
 
 # Setup and build frontend
-RUN cd frontend && npm install && npm run build
+RUN cd frontend && npm install --ignore-scripts && npm run build
```

Comment:
Why would we ignore scripts when we can run preinstall and postinstall scripts? I am okay with this change in general, but let us add a dummy script in package.json with key `README` and value something like `DOCKERFILE NPM INSTALL IGNORES SCRIPTS` so that in the future it will make debugging easier if someone does intend to add a pre/post install script to the manifest.

Reply:
@eternaltyro  Hey, i had these modified as part of security recommendation from sonar cube. i had these tested as well. But looks like /scripts/docker/tasking-manager/Dockerfile is replaced by Dockerfile.backend & Dockerfile.frontend resp. We should be looking to remove this in future.

Reply:
Good for now.